_dev/src/ts/pages/UpdatePageUpdate.ts

```diff
@@ -1,13 +1,61 @@
 import UpdatePage from './UpdatePage';
+import ProgressTracker from '../components/ProgressTracker';
+import { ApiResponseAction } from '../types/apiTypes';
+import Process from '../utils/Process';
+import api from '../api/RequestHandler';
 
 export default class UpdatePageUpdate extends UpdatePage {
   protected stepCode = 'update';
+  #progressTracker: ProgressTracker = new ProgressTracker(this.#progressTrackerContainer);
 
   constructor() {
     super();
   }
 
-  public mount() {
+  public mount = async () => {
     this.initStepper();
+
+    const updateAction = 'UpdateInitialization';
+
+    const process = new Process({
+      onProcess: this.#onProcess,
+      onProcessEnd: this.#onProcessEnd,
+      onError: this.#onError
+    });
+
+    await process.startProcess(updateAction);
+  };
+
+  public beforeDestroy = () => {
+    this.#progressTracker.beforeDestroy();
+  };
+
+  get #progressTrackerContainer(): HTMLDivElement {
+    const progressTrackerContainer = document.querySelector(
+      '[data-component="progress-tracker"]'
+    ) as HTMLDivElement;
+
+    if (!progressTrackerContainer) {
+      throw new Error('Progress tracker container not found');
+    }
+
+    return progressTrackerContainer;
```

Comment:
Any reason for not using the `queryElement`?

```suggestion
    return this.queryElement<HTMLTemplateElement>(
      '[data-component="progress-tracker"]'
      'Progress tracker container not found'
    );
```

If this is because it does not extends `ComponentAbstract`, it could be nice to make PageAbstract extending it to get this feature.
This would make some changes on all existing pages, this can be done in another PR.

Reply:
Yes I going to check this in another PR